assignments/arrays.js

```diff
@@ -63,12 +63,16 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+
+for (let i=0; i < inventory.length; i++){
+    inventory[i];
+};
```

Comment:
For this first challenge a for loop is not needed since you just want one object of the array. However it seems like you didn't use it so maybe just forgot to delete it